cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -218,8 +221,7 @@ public ServiceTicket grantServiceTicket(
                     uniqueTicketIdGenKey);

```

Comment:
Most checks are the same for either call, because PT really is an ST. So checks that deal with service registry and authn policy and such are all shared. I'll take a look at the proxiedBy field and see if that can be moved elsewhere. Not sure yet.